src/Cli/dotnet/commands/dotnet-sln/list/Program.cs

```diff
@@ -2,23 +2,29 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.CommandLine;
+using System.Text.Json;
 using Microsoft.DotNet.Cli;
 using Microsoft.DotNet.Cli.Sln.Internal;
 using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.Tools.Common;
+using Microsoft.DotNet.Tools.Tool.List;
 
 namespace Microsoft.DotNet.Tools.Sln.List
 {
     internal class ListProjectsInSolutionCommand : CommandBase
     {
+        private readonly static JsonSerializerOptions s_noEscapeJsonSerializerOptions = new() { Encoder = System.Text.Encodings.Web.JavaScriptEncoder.UnsafeRelaxedJsonEscaping };
```

Comment:
Out of curiosity, why use this encoder specifically?

Reply:
I got confused and thought that this was whar `dotnet list package` used. Is there a different one that is more common/would not pique your curiosity? :P

Reply:
I don't actually know what they use (the implementation is over in NuGet.Client repo I suppose, I could go look), but if you aligned with there decision I'm 100% ok with using that!